### PR TITLE
docs: update chinese translation

### PR DESCRIPTION
**Pull request by AnsonZnl**
Closes #<issue>

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

Update faq Chinese document

---

## Screenshots

_[Screenshots]_

💯

### CODE REVIEW

File: www/src/pages/zh-hans/faq.mdx
Comment (fernandoxu) [inline www/src/pages/zh-hans/faq.mdx:1]: isMdx: true
---

Comment (fernandoxu) [inline www/src/pages/zh-hans/faq.mdx:1]: isMdx: true
---

Comment (fernandoxu) [inline www/src/pages/zh-hans/faq.mdx:1]: import Callout from "../../components/docs/callout.tsx";

Comment (fernandoxu) [inline www/src/pages/zh-hans/faq.mdx:1]: <Callout type="info">
  对于 `/app` 目录中支持的、计划的和正在开发的功能列表,请访问 [beta Next.js
  docs](https://beta.nextjs.org/docs/app-directory-roadmap#supported-and-planned-features).
</Callout>

**Review by fernandoxu — CHANGES_REQUESTED** [review]
sorry I mean we should sync with [`www/src/pages/en/faq.mdx`](https://github.com/t3-oss/create-t3-app/blob/next/www/src/pages/en/faq.mdx),  you can follow my comments and update the code.

**Review by fernandoxu — APPROVED** [review]

**Review by escwxyz — APPROVED** [review]

**Review by c-ehrlich — APPROVED** [review]

**Pull request merged**